src/main/java/org/sagebionetworks/bridge/config/Environment.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.config;
+
+public enum Environment {
+    LOCAL,
+    DEV,
+    DEVELOP,
+    DEVELOPMENT,
+    STAGING,
+    UAT,
+    PROD,
+    PRODUCTION
```

Comment:
I agree with Alx here. We should either enforce the canonical local, dev, uat, prod, or we don't enforce it at all and just use an arbitrary string. Having synonyms for the different envs will cause more confusion than the flexibility is worth.